go/coordinator/cmd/grpccoordinator/cmd.go

```diff
@@ -0,0 +1,38 @@
+package grpccoordinator
```

Comment:
Can we add a README for how to run this / high level description of key components